addon/validators/date.js

```diff
@@ -53,9 +59,11 @@ export default Base.extend({
   },
 
   validate(value, options) {
+    options = assign({}, options);
```

Comment:
Yep your right, I thought it shouldn't need it. The tests are mutating the options because processOptions isn't called. I need to clone the object in the tests. I'll get that fixed.